packages/dev/core/src/Physics/v2/Plugins/havokPlugin.ts

```diff
@@ -2267,10 +2271,14 @@ export class HavokPlugin implements IPhysicsEnginePluginV2 {
      * Dispose the world and free resources
      */
     public dispose(): void {
-        this._hknp.HP_QueryCollector_Release(this._queryCollector);
-        this._queryCollector = BigInt(0);
-        this._hknp.HP_World_Release(this.world);
-        this.world = undefined;
+        if (this._queryCollector) {
+            this._hknp.HP_QueryCollector_Release(this._queryCollector);
+            this._queryCollector = BigInt(0);

```

Comment:
if this is the incorrect type, why not set it to undefined?

Reply:
Replaced with `undefined`